.gitignore

```diff
@@ -23,7 +23,6 @@ composer.lock
 # Project Specific
 node_modules
 *.log
-lib
```

Comment:
Revert this, and remove all the `lib/*` files you've added in this PR. They are build output, and never under version control.